internal/drivers/anka/pool.go

```diff
@@ -0,0 +1,237 @@
+package anka
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/drone-runners/drone-runner-aws/internal/lehelper"
+	"github.com/drone-runners/drone-runner-aws/types"
+	"github.com/drone/runner-go/logger"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	ankabin = "/usr/local/bin/anka"
+)
+
+type ankaShow struct {
+	UUID    string `json:"uuid"`
+	Name    string `json:"name"`
+	Created string `json:"created"`
+	Memory  string `json:"memory"`
+	IP      string `json:"ip"`
+}
+
+func (p *provider) RootDir() string {
+	return p.rootDir
+}
+
+func (p *provider) ProviderName() string {
+	return string(types.ProviderAnka)
+}
+
+func (p *provider) Ping(_ context.Context) error {
+	return nil
```

Comment:
ping is needed

Reply:
ping what? The app is installed or its not.

Reply:
there's nothing to "ping"

Reply:
as a bare minimum i would expect we check that the binary for anka is available.

Reply:
done